modules/check_in/spec/requests/check_in/v0/travel_claims_spec.rb

```diff
@@ -74,5 +75,37 @@
         expect(response.body).to be_blank
       end
     end
+
+    context 'when session is authorized and travel claim submitted oh facility type' do
```

Comment:
what value is this spec adding?

Reply:
I thought this would cover all possible ways of calling the submission worker.

Reply:
@randomsync can you remove `facility_type` from the controller params if it's only being used in the logging? 

https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/check_in/app/controllers/check_in/v0/travel_claims_controller.rb#L18